### ISSUE TITLE
White background

### ISSUE THREAD

**Issue by vogloblinsky** [parkerbennett/stackicons#5]
If icon used in a non-white background website, there isn't any background due to icon structure.

**Comment by parkerbennett** [comment]
I wanted to limit the colors to four to keep the non-semantic <span>s down to one. So, as a cheat I rely on a light background for some icons, e.g., the whites of the eyes 
in the GitHub octocat. Stackicons-Social has a lot of flexibility, though. If you look through the character set. You could swap out the octocat “face” that has eyes cut out for one that doesn’t so the background doesn’t show through, or you could add an extra <span> and render the silhouette of the octocat in white behind everything.

The colors can be swapped out easily in the SASS-based construction kit files, as well.

If you have a specific icon you need modified let me know and I’ll see if I can help.

Parker

**Comment by cognitom** [comment]
How about this approach? @RoelN uses :first-letter and :first-line.
https://github.com/RoelN/multicolor-icons-demo/blob/master/skwirrol-icons.css

**Comment by parkerbennett** [comment]
Yes, I’ve been meaning to check this out more thoroughly, thanks.
No IE8 support, but it might be worth it.

**Comment by RoelN** [comment]
IE8 support is just a matter of falling back to EOT. It's not included in my demo, but can be simply done by adding it to the <code>@font-face</code> rule!
